src/main/java/com/github/manolo8/darkbot/modules/utils/NpcAttacker.java

```diff
@@ -110,7 +110,7 @@ protected void tryAttackOrFix() {
             sendAttack(250, 5000, true);
         } else if (!hero.isAttacking(target) || !hero.isAiming(target)) {
             sendAttack(1500, 5000, false);
-        } else if (target.health.hpDecreasedIn(1500) || hero.locationInfo.distance(target) > 700) {
+        } else if ((target.health.hpDecreasedIn(1500) || target.hasEffect(16)) || hero.locationInfo.distance(target) > 700) {
```

Comment:
Avoid using magic numbers, use the effect constants when possible target.hasEffect(Effect.NPC_ISH)

Reply:
paranthesis was there to specify ish check is meant to work together with hp check which isn't related to distance, helps reader by being more understandable for future
if you agree i'll change to effect name only

Reply:
https://github.com/darkbot-reloaded/DarkBot/pull/49/commits/22e09ec75c29ed4437f946bc19509667c9592bc1